docs/user/deploying-feeds.md

```diff
@@ -0,0 +1,34 @@
+# Deploying GTFS Feeds to OTP
+
+Assumptions:
+
+* [X] You have loaded a GTFS feed into a project.
+* [X] You have a deployment server available.
```

Comment:
Shouldn't another assumption be that an osm-lib server has been setup for datatools to use in order to fetch OSM data?

Reply:
Oh. Do we have anything that talks about setting up an osm-lib server?

Reply:
Yeah. It'd be fine to have a single line with a checkbox that just links to the repository page which is here: https://github.com/conveyal/osm-lib